src/github.com/matrix-org/dendrite/roomserver/storage/storage.go

```diff
@@ -38,7 +38,7 @@ func (d *Database) SetPartitionOffset(topic string, partition int32, offset int6
 }
 
 // StoreEvent implements input.EventDatabase
-func (d *Database) StoreEvent(event gomatrixserverlib.Event) error {
+func (d *Database) StoreEvent(event gomatrixserverlib.Event, authEventNIDS []int64) error {
```

Comment:
From an API perspective, if feels weird that the auth event NIDs aren't bundled with the `event`. Why is this?